lib/krane/kubernetes_resource/daemon_set.rb

```diff
@@ -3,7 +3,7 @@
 module Krane
   class DaemonSet < PodSetBase
     TIMEOUT = 5.minutes
-    SYNC_DEPENDENCIES = %w(Pod)
+    SYNC_DEPENDENCIES = %w(Pod.apps)
```

Comment:
Having to add a group here sucks. I'm need to look at the behavior for kubectl and see what group it picks if there are multiple for a kind 🤞 its not the one a crd defines...